autodispose/src/main/java/com/uber/autodispose/CompletableSubscribeProxy.java

```diff
@@ -60,4 +61,21 @@
    * @return a {@link CompletableObserver}
    */
   @CheckReturnValue <E extends CompletableObserver> E subscribeWith(E observer);
+
+  /**
+   * Creates a TestObserver and subscribes
+   * it to this Completable.
+   *
+   * @return a {@link TestObserver}
+   */
+  @CheckReturnValue TestObserver<Void> test();
```

Comment:
Can you change these javadocs to match the others and link the corresponding methods they're mirroring?

Reply:
Alright!